recipes/openai-whisper/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "openai-whisper" %}
+{% set version = "20231117" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openai-whisper-{{ version }}.tar.gz
+  sha256: 7af424181436f1800cc0b7d75cf40ede34e9ddf1ba4983a910832fcf4aade4a4
+
+build:
+  entry_points:
+    - whisper=whisper.transcribe:cli
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+  skip: true  # [win]
```

Comment:
I could not get this to build on Windows because of pytorch

Reply:
If a dependency is not available on a specific OS, we prefer to just let it fail instead of skipping it. Once the dependency becomes available, the package can be installed there as well.